### DISCUSSION TITLE
Block #ID ?

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#34] [Q&A] [answered]
Hi again
What is this ID for? 

![grafik](https://github.com/user-attachments/assets/c12a2756-4084-40eb-99bf-b1a65fed2021)

Thx

**Comment by marcantondahmen** [comment, marked as answer]
Hi, you can define a custom id attribute for that block that can be used as target for links.

**Comment by cracksilver** [comment]
ah ok. It's like an anchor. And then you can point to it in the URL for example #1 https://mydomain.com/product1#1
